### ISSUE TITLE
systemd-resolve has moved to resolvectl on ubuntu 22.04, which causes `kubevpn connect` fail

### ISSUE THREAD

**Issue by szu17dmy** [kubenetworks/kubevpn#335]
There is not systemd-resolve on ubuntu 22.04, we may need to use resolvectl instead.

```
Failed to exec cmd: systemd-resolve --set-dns 10.10.129.161 --interface utun0 --set-domain=default.svc.cluster.local --set-domain=svc.cluster.local --set-domain=cluster.local, message: , ignore
detected 127.0.0.53 nameserver, assuming systemd-resolved, so using resolv.conf: /run/systemd/resolve/resolv.conf
Configured DNS service
```

After executing thoes commands instead, cluster internal url will become available.

``` shell
resolvectl dns utun0 10.10.129.161
resolvectl domain utun0 default.svc.cluster.local svc.cluster.local cluster.local
```

**Comment by wencaiwulue** [comment]
no... it's a bug

**Comment by szu17dmy** [comment]
How should we fix it? Update `SetupDNS` in dns_linux.go by adding resolvectl or use nmcli instead?

**Comment by wencaiwulue** [comment]
> How should we fix it? Update `SetupDNS` in dns_linux.go by adding resolvectl or use nmcli instead?

i don't find good solution to solve this problem, maybe we can use this library https://github.com/coreos/go-systemd? to communicate with systemd directlly?

**Comment by wencaiwulue** [comment]
@szu17dmy i fixed it by pr #336 can you help to test it ? thans in advance

**Comment by wencaiwulue** [comment]
@szu17dmy already fixed it, you can use version v2.2.19 ~
